### ISSUE TITLE
path to snippets is added in the beginning *yas-snippet-dirs*

### ISSUE THREAD

**Issue by jgrzebyta** [mrkkrp/common-lisp-snippets#2]
After loading common-lisp-snippets variable **yas-snippet-dirs** looks like:

```
("/home/jacek/.emacs.d/elpa/common-lisp-snippets-20150603.944/snippets" "~/.emacs.d/snippets" yas-installed-snippets-dir)
```

In that way all custom snippets go into common-lisp-snippets directory. 

The value of **yas-installed-snippets-dir** variable is also a path into the module snipps. Maybe it is not necessary to add the snippets path manually in autoload.

My current versions are:
emacs 24.4.1
common-lisp-snippets 20150603.944
yasnippet 20150811.1222

**Comment by mrkkrp** [comment]
Hi, @jgrzebyta, thank you for the feedback. This should have been fixed in 3b2b50f. Wait a bit till MELPA catches up. If the problem persists feel free to reopen.

**Comment by jgrzebyta** [comment]
Hi,

Now it's fixed. Thank you.

Jacek
